test/test.hs

```diff
@@ -1,5 +1,6 @@
 import Prelude ()
 
+import           Data.List (inits)
```

Comment:
I think it's fine if we import `Game.LambdaHack.Core.Prelude` instead, as in all other modules.